### PR TITLE
feat(Hamburger menu) : implement hamburger menu for mobile

### PR DESCRIPTION
**Pull request by kibo-susantasamanta**
@kibo-kevinwatts @kibo-sushant @Chandradeepta

### CODE REVIEW

File: components/KiboHamburgerMenu.vue [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+      gotoProduct,
+    }
+  },
+  data() {
+    return {
+      position: "left",
+      staticClass: null,
+      className: null,
+      active: "",
+      title: "Back",
```
Comment (kibo-chandradeeptalaha) [inline components/KiboHamburgerMenu.vue]: Already being passed from the parent. Either use this or use the parent props.

File: components/KiboAppHeader.vue [hunk 2]
```diff
@@ -1,11 +1,66 @@
...
+      <template #content-top>
+        <SfButton
+          class="sf-button--pure sf-header__action sf-header__bottom-link sf-header__account"
+          @click="handleAccountClick"
+        >
+          <SfIcon class="sf-header__icon">
+            <font-awesome-icon :icon="[accountIcon, 'user-circle']" class="fa-icon" />
+          </SfIcon>
+          <div class="icon-name-sidebar">
+            <span> My Account</span>
```
Comment (kibo-kevinwatts) [inline components/KiboAppHeader.vue]: Use the lang en.js file, don't hardcode `My Account`

File: components/KiboHamburgerMenu.vue (hunk 1 above)
Comment (kibo-susantasamanta) [reply components/KiboHamburgerMenu.vue]: fixed.

File: components/KiboAppHeader.vue (hunk 2 above)
Comment (kibo-susantasamanta) [reply components/KiboAppHeader.vue]: Resolved.

**Review by kibo-kevinwatts — CHANGES_REQUESTED** [review]
Check comments on jira icky-402 for changes

**Review by kibo-kevinwatts — APPROVED** [review]
This looks good!

**Pull request merged**